src/main/java/com/ixi_U/user/dto/CreateSubscribedRequest.java

```diff
@@ -0,0 +1,10 @@
+package com.ixi_U.user.dto;
+
+import jakarta.validation.constraints.NotBlank;
+
+public record CreateSubscribedRequest(
+        @NotBlank
+        String planId
+) {
```

Comment:
```suggestion
public record CreateSubscribedRequest(@NotBlank(message = "유효하지 않은 ID") String planId) {
```
- 가로 방향으로 글자 수가 너무 많은 것이 아니라면 불필요한 줄바꿈을 삼가는게 어떨까요?
- 검증에 걸릴 경우 예외 상황을 처리하기 위해 메시지를 포함하는 것은 어떨까요?

Reply:
수정했습니다.